custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -0,0 +1,35 @@
+@import "app";
```

Comment:
most of this commit was brought in from https://github.com/dimagi/mobile-dashboard-prototype/pull/16 with no changes